### PR TITLE
feat(attributes): new repo name attributes (short and full)

### PR DESCRIPTION
**Pull request by ErwanSimonetti**
Adding new attributes `repository-name-short` and `repository-name-full`

Fixes MRGFY-905

### CODE REVIEW

**Comment by ErwanSimonetti** [comment]
I'm just missing the release-note because I can't find the exact `reno` command i'm supposed to use

**Review by jd — CHANGES_REQUESTED** [review]
Name update plz 😁

File: mergify_engine/context.py [hunk 1]
```diff
@@ -795,6 +795,14 @@ def base_url(self) -> str:
         # TODO(sileht): remove me when context split if done
         return self.repository.base_url
 
+    @property
```
Comment (sileht) [inline mergify_engine/context.py:798]: We don't need these new properties, they don't add much value.

File: mergify_engine/context.py [hunk 2]
```diff
@@ -1337,6 +1345,10 @@ async def _get_consolidated_data(self, name: str) -> ContextAttributeType:
                 for t in await self.retrieve_review_threads()
                 if not t["isResolved"]
             ]
+        elif name == "repository-name":
+            return self.repo_name
```
Comment (sileht) [inline mergify_engine/context.py:1349]: You can directly use `self.repository.repo["name"]`

**Review by sileht — CHANGES_REQUESTED** [review]
Everything looks good to me. Just a couple of lines I think we can remove.

File: mergify_engine/context.py [hunk 3]
```diff
@@ -795,6 +795,14 @@ def base_url(self) -> str:
...
         return self.repository.base_url
 
+    @property
+    def repo_name(self) -> str:
+        return self.repository.repo["name"]
+
+    @property
+    def repo_full_name(self) -> str:
+        return self.repository.repo["full_name"]
+
```
Comment (jd) [inline mergify_engine/context.py:805]:
```suggestion
```

File: mergify_engine/context.py (hunk 2 above)
Comment (jd) [inline mergify_engine/context.py:1349]:
```suggestion
            return self.repository.repo["name"]
```

File: mergify_engine/context.py [hunk 4]
```diff
@@ -1337,6 +1345,10 @@ async def _get_consolidated_data(self, name: str) -> ContextAttributeType:
                 for t in await self.retrieve_review_threads()
                 if not t["isResolved"]
             ]
+        elif name == "repository-name":
+            return self.repo_name
+        elif name == "repository-full-name":
+            return self.repo_full_name
```
Comment (jd) [inline mergify_engine/context.py:1351]:
```suggestion
            return self.repository.repo["full_name"]
```

**Review by jd — CHANGES_REQUESTED** [review]

**Review by jd — APPROVED** [review]

**Pull request merged**